lib/api/routes/iterations.rb

```diff
@@ -54,7 +54,9 @@ class Iterations < Core
         end
         opts = {
           code_analysis: CodeAnalyzer.build({ language: data["language"],
-                                              code: data["code"] }).run,
+                                              code: data["code"],
+                                              git_rep_info: user.username+"/"+data['problem'],
```

Comment:
Line is too long. [94/80]<br>Surrounding space missing for operator '+'.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.